update/delta/delta_1.3.6_add_vl_for_SIA_export.sql

```diff
@@ -0,0 +1,237 @@
+UPDATE qwat_vl.status SET sia405code = 'weitere.Projekt' WHERE value_fr = 'projet en execution';
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.subscriber_type SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.survey_type SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.tank_firestorage SET sia405code = 'unbekannt' WHERE ???;
+
+-- TODO : valve function actuation complete [X] to check [ ]
+UPDATE qwat_vl.valve_actuation SET sia405code = 'unbekannt' WHERE value_fr = 'autre';
+UPDATE qwat_vl.valve_actuation SET sia405code = 'unbekannt' WHERE value_fr = 'inconnu';
+UPDATE qwat_vl.valve_actuation SET sia405code = 'unbekannt' WHERE value_fr = 'à déterminer';
+UPDATE qwat_vl.valve_actuation SET sia405code = 'mechanisch' WHERE value_fr = 'manuel';
+UPDATE qwat_vl.valve_actuation SET sia405code = 'mechanisch' WHERE value_fr = 'manuel à gauche';
+UPDATE qwat_vl.valve_actuation SET sia405code = 'motorisch.unbekannt' WHERE value_fr = 'électrique';
+UPDATE qwat_vl.valve_actuation SET sia405code = 'motorisch.mit_Fernsteuerung' WHERE value_fr = 'télécommandée';
+UPDATE qwat_vl.valve_actuation SET sia405code = 'motorisch.unbekannt' WHERE value_fr = 'automatique';
+
+-- TODO : valve function mapping complete [X] to check [ ]
+UPDATE qwat_vl.valve_function SET sia405code = 'Schieber.unbekannt' WHERE value_fr = 'autre';
+UPDATE qwat_vl.valve_function SET sia405code = 'Schieber.unbekannt' WHERE value_fr = 'inconnu';
+UPDATE qwat_vl.valve_function SET sia405code = 'Schieber.unbekannt' WHERE value_fr = 'à déterminer';
+UPDATE qwat_vl.valve_function SET sia405code = 'Schieber.Zonenschieber' WHERE value_fr = 'vanne de régulation';
+UPDATE qwat_vl.valve_function SET sia405code = 'Schieber.unbekannt' WHERE value_fr = 'ventouse';
+UPDATE qwat_vl.valve_function SET sia405code = 'Schieber.unbekannt' WHERE value_fr = 'vanne bypass';
+UPDATE qwat_vl.valve_function SET sia405code = 'Schieber.Anschlussschieber' WHERE value_fr = 'vanne branchement';
+UPDATE qwat_vl.valve_function SET sia405code = 'Schieber.unbekannt' WHERE value_fr = 'vanne incendie';
+UPDATE qwat_vl.valve_function SET sia405code = 'Schieber.Hydrantenschieber' WHERE value_fr = 'vanne d''hydrant';
+UPDATE qwat_vl.valve_function SET sia405code = 'Schieber.Entleerungsschieber' WHERE value_fr = 'vidange';
+UPDATE qwat_vl.valve_function SET sia405code = 'Schieber.Trennschieber' WHERE value_fr = 'vanne réseau';
+UPDATE qwat_vl.valve_function SET sia405code = 'Schieber.Entleerungsschieber' WHERE value_fr = 'vidange-ventouse';
+UPDATE qwat_vl.valve_function SET sia405code = 'Schieber.unbekannt' WHERE value_fr = 'antiretour';
+UPDATE qwat_vl.valve_function SET sia405code = 'Schieber.unbekannt' WHERE value_fr = 'vanne sprinkler';
+UPDATE qwat_vl.valve_function SET sia405code = 'Schieber.unbekannt' WHERE value_fr = 'jauge';
+
+-- TODO : complete mapping
+-- UPDATE qwat_vl.valve_maintenance SET sia405code = 'unbekannt' WHERE ???;
```

Comment:
Just copy value to Absperrorgan.Zustand (TEXT*30)

Reply:
+1 @olivierdalang ;)